edx_event_bus_kafka/internal/consumer.py

```diff
@@ -221,10 +233,11 @@ def report_receiver_errors(self, send_results):
             except AttributeError:
                 receiver_name = str(receiver)
 
+            # The stack traces are already logged by django.dispatcher, so just the error message is fine.
```

Comment:
@timmc-edx: I had the thought that the ReceiverError below could include something like "See separate django.dispatcher log messages for stack traces."

Reply:
Added some information to https://2u-internal.atlassian.net/wiki/spaces/AT/pages/234258433/Event+Bus+alerts about the split.

I wouldn't be totally opposed to "(stacktrace elsewhere in logs)" or something equally short. We'd want a unit test to ensure that it stays true, though.

Reply:
If we want to be fast about it, we could consider: https://github.com/openedx/event-bus-kafka/pull/82/files#r1025813178. Otherwise, maybe the test is really fast. Otherwise, we should skip and move on.

Reply:
Here's a PR to do this: https://github.com/openedx/event-bus-kafka/pull/84